src/CommonAssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 
 [assembly: ComVisible(false)]
 
-[assembly: AssemblyVersion("2.11.2.0")]
-[assembly: AssemblyFileVersion("2.11.2.0")]
+[assembly: AssemblyVersion("2.11.3.0")]
+[assembly: AssemblyFileVersion("2.11.3.0")]
```

Comment:
Should nuget.server just move to 2.14.0 to match nuget.core?  These packages use to be released together. Either way is fine, it just seems like we could simplify things here.